modules/runner/src/integrationTest/java/org/apache/ignite/internal/distribution/zones/ItDistributionZonesFilterTest.java

```diff
@@ -220,15 +199,15 @@ void testFilteredEmptyDataNodesDoNotTriggerRebalance() throws Exception {
         TablePartitionId partId = new TablePartitionId(table.tableId(), 0);
 
         // Table was created after both nodes was up, so there wasn't any rebalance.
-        assertPendingStableAreNull(metaStorageManager, partId);
+        assertPendingsAreNull(metaStorageManager, partId);
```

Comment:
This method was needed to check there wasn't any rebalance at all, now we check only pending, is this correct to check like this the absence of a rebalance?